pkg/stockings/stock.go

```diff
@@ -0,0 +1,195 @@
+package stockings
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+/*
+	Could look to make this an interface as we move forward with different stock apis
+	ie make different structs for td, iex, rh apis and implement each of these methods for the
+	different apis via StockAPI interface.
+
+*/
+type IexStock struct {
+	Symbol        string  `json:"symbol"`
+	Name          string  `json:"companyName"`
+	LatestPrice   float32 `json:"latestPrice"`
+	Change        float32 `json:"change"`
+	ChangePercent float32 `json:"changePercent`
+}
+
+type IexHistoricalStock struct {
+	Date  string  `json:"date"`
+	Price float32 `json:"close"`
+}
+
+type IexHistoricalStocks []IexHistoricalStock
+
+const (
+	iexCurrentPriceUrl        = "https://cloud.iexapis.com/stable/stock/%s/quote?token=%s"
+	iexHistoricalDateRangeUrl = "https://cloud.iexapis.com/stable/stock/%s/chart/%s?token=%s"
+	iexHistoricalDateUrl      = "https://cloud.iexapis.com/stable/stock/%s/chart/date/%s?chartByDay=true&token=%s"
+	//https://cloud.iexapis.com/stable/stock/MELI/chart/date/20200102?chartByDay=true&token=pk_ec21611ca5f5492e9397b4a1879ff114
```

Comment:
what is this comment for?

Reply:
ralles made this when creating the url format